MailCore/Models/CommonContact.swift

```diff
@@ -46,14 +46,117 @@ public struct AvatarImageRequest {
     }
 }
 
-public struct CommonContact {
+public enum CommonContactBuilder {
```

Comment:
I'm afraid the name `CommonContactBuilder` might be confusing, one could imagine that it is a resultBuilder

Reply:
Good point, so as something that wrapps all the parameters to build a contact.
Some LLM I wil not name did propose this as names (skip the first two):

    CommonContactBuilder
    ContactBuilder
    CommonContactConstructor
    ContactParametersWrapper
    ContactSetup
    ContactConfiguration

@valentinperignon Which one do you prefer ?

Reply:
Discussed, used ContactConfiguration